apps/fabric-website/src/components/App/AppState.tsx

```diff
@@ -62,7 +62,7 @@ export const AppState: IAppState = {
         },
         {
           title: 'Brand icons',
-          url: '#/styles/brand-icons',
+          url: '#/styles/brandicons',
```

Comment:
Discussed in person--while it does indeed make sense for these to be consistent, doing this change will also break existing links. There may be a better way to handle a change/transition like this that won't result in a break--but we should probably look at doing that in a separate PR. That said--thanks for catching this!